wp-content/themes/midwestenergynews/less/archive.less

```diff
@@ -279,3 +279,11 @@ body.category-national {
       color: @black;
   }
 }
+
+.archive.category-digest {
+  #sidebar {
```

Comment:
Should this be applied to the homepage as well, or all pages?

Reply:
I don't think this one should be since it would modify every `btn-primary` on their site, which they might not want.